installer.sh.in

```diff
@@ -985,10 +1033,12 @@ the mount point for the root filesystem (/) has not yet been configured." ${MSGB
         DIALOG --msgbox "${BOLD}${RED}ERROR:${RESET} \
 /usr mount point has been configured but is not supported, please remove it to continue." ${MSGBOXSIZE}
         return 1
-    elif [ -n "$EFI_SYSTEM" -a -z "$efi_system_partition" ]; then
+    elif [ -n "$EFI_SYSTEM" -a \( -z "$efi_system_partition" \) ]; then
```

Comment:
Why?

Reply:
Because before I made that change, that conditional would return TRUE, even if the $efi_system_partition variable was set